samples/matter/common/src/bt_nus_service.cpp

```diff
@@ -85,6 +90,10 @@ void NUSService::StartServer()
 		return;
 	}
 
+	VerifyOrReturn(bt_conn_auth_cb_register(&sConnAuthCallbacks) == 0);
```

Comment:
Maybe we should change the type returned by `StartServer` to bool or int? So far there wasn't much that could fail in this method, but now we've got some checks moved from `Init` that actually may fail.